src/DynamicEvaluator/Application/Checker/ClassExtendsUpdatedPackageChecker/ClassExtendsUpdatedPackageChecker.php

```diff
@@ -19,7 +19,7 @@ class ClassExtendsUpdatedPackageChecker
     /**
      * @var string
      */
-    protected const ERROR_MESSAGE = 'Project class extends class from updated package';
+    protected const ERROR_MESSAGE = 'These classes of yours extend modified Spryker Core classes and need to be checked';
```

Comment:
I would rather write: 

The following classes you've created extend modified Spryker Core classes and require verification.

But if it's a PM requirement, then it's fine.

Reply:
PM Figma requirements